.github/workflows/release-pypi.yml

```diff
@@ -74,7 +51,7 @@ jobs:
 
     - name: Build packages
       env:
-        PACKAGE_NAME: ${{ matrix.pkg }}
+        PACKAGE_NAME: ${{ matrix.pkg-name }}
       run: |
         python setup.py sdist bdist_wheel
```

Comment:
Before we start building and uploading and all, we still need to get `setup_tools.py` from LAI branch to master

Reply:
Yes, this needs to be merged after #15105